app/View/Label.php

```diff
@@ -158,8 +158,8 @@ public function render(callable $callback = null)
                                 // The end result of this will be in this format:
                                 // {labelOne} {valueOne} | {labelTwo} {valueTwo} | {labelThree} {valueThree}
                                 $previous['value'] = trim(implode(' | ', [
-                                    implode(' ', [$previous['label'], $previous['value']]),
-                                    implode(' ', [$current['label'], $current['value']]),
```

Comment:
Was this supposed to be included?

Reply:
whoops those are changes for something else.